index.html

```diff
@@ -263,25 +263,25 @@ <h3 class="connect">
         <div class="language-header">
           <li>Languages</li>
           <li>
-            <img class="toggle" src="./Graphics/Normal Button/Tertiary/Icons/toggle.svg" alt="" />
+            <img class="toggle" src="./Graphics/Normal Button/Tertiary/Icons/toggle.svg" alt="toggle-languages" />
           </li>
         </div>
 
         <ul class="languages">
 
           <div class="fordesk">
             <div class="js-container">
-              <img src="./Graphics/Normal Button/Tertiary/Icons/javascript.png" alt="" />
+              <img src="./Graphics/Normal Button/Tertiary/Icons/javascript.png" alt="javascript-language-icon" />
               <li class="javascript">JavaScript</li>
```

Comment:
- [ ] You have done great having the `languages, frameworks and skills` titles 👏 . However, for [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) purposes I would suggest using [HTML headings](https://www.w3schools.com/html/html_headings.asp). This increases the accessibility of the website to screen readers. Read through this [Stackoverflow conversation](https://stackoverflow.com/questions/6555014/is-it-semantically-incorrect-to-place-h2-tags-between-li-tags) to have a clear understanding ☺️ .
 
- The design 😎 
![head](https://user-images.githubusercontent.com/99394374/214267459-d61a64d4-5a7d-4ff9-bb9c-c4671b751c65.PNG)